.changeset/pre.json

```diff
@@ -20,6 +20,7 @@
     "@solana/web3.js": "0.0.0-development",
     "@solana/options": "2.0.0-preview.2",
     "@solana/programs": "2.0.0-preview.2",
+    "@solana/react": "2.0.0-preview.3",
```

Comment:
Should this be `3`?

Reply:
They should all be 3. I forgot to advance these again after the last publish.